src/store/index.js

```diff
@@ -1,10 +1,25 @@
 import { createStore } from 'redux';
 import thunkMiddleware from 'redux-thunk';
 
-const reducer = (users, action) {
-    
+const reducer = (users, action) => {
```

Comment:
Reducer has to be exported for testing. And it basically should be total oposit of this code:
        switch(action.type){
           case 'FETCH_USER_START':
               return {....state, fetching: true}
}
and so on